examples/cloud/chatgpt_apps/timer/README.md

```
@@ -0,0 +1,224 @@
+```
+
+Spin up the mcp-agent server locally with SSE transport:
+
+```bash
+uv run main.py
+```
+
+This will:
+
+- Start the MCP server on port 8000
+- Serve the web client at http://127.0.0.1:8000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)